### ISSUE TITLE
NullPointerException related to voice messages on startup, version 5.20.1

### ISSUE THREAD

**Issue by link2xt** [signalapp/Signal-Android#11526]
- [x] I have searched open and closed issues for duplicates
- [x] I am submitting a bug report for existing functionality that does not work as intended
- [x] <del>I have read https://github.com/signalapp/Signal-Android/wiki/Submitting-useful-bug-reports</del>There is no such page.
- [x] This isn't a feature request or a discussion topic

----------------------------------------

### Bug description

Signal version 5.20.1 installed from Google Play Beta channel crashes on startup.

### Steps to reproduce

- start the application
- see "the application was closed, ..." window

### Device info

**Android version:** 8.1.0
**Signal version:** 5.20.1 (Google Play Beta)

### Debug log

<details>
<summary>logcat crash log</summary>

```
--------- beginning of crash
08-06 20:28:30.257  5493  5493 E AndroidRuntime: FATAL EXCEPTION: main
08-06 20:28:30.257  5493  5493 E AndroidRuntime: Process: org.thoughtcrime.securesms, PID: 5493
08-06 20:28:30.257  5493  5493 E AndroidRuntime: java.lang.NullPointerException: sensorManager.getDefault…or(Sensor.TYPE_PROXIMITY) must not be null
08-06 20:28:30.257  5493  5493 E AndroidRuntime:        at org.thoughtcrime.securesms.components.voice.VoiceNoteProximityWakeLockManager.<init>(VoiceNoteProximityWakeLockManager.kt:39)
08-06 20:28:30.257  5493  5493 E AndroidRuntime:        at org.thoughtcrime.securesms.components.voice.VoiceNoteMediaController$ConnectionCallback.onConnected(VoiceNoteMediaController.java:280)
08-06 20:28:30.257  5493  5493 E AndroidRuntime:        at android.support.v4.media.MediaBrowserCompat$ConnectionCallback$StubApi21.onConnected(MediaBrowserCompat.java:669)
08-06 20:28:30.257  5493  5493 E AndroidRuntime:        at android.support.v4.media.MediaBrowserCompatApi21$ConnectionCallbackProxy.onConnected(MediaBrowserCompatApi21.java:103)
08-06 20:28:30.257  5493  5493 E AndroidRuntime:        at android.media.browse.MediaBrowser$6.run(MediaBrowser.java:603)
08-06 20:28:30.257  5493  5493 E AndroidRuntime:        at android.os.Handler.handleCallback(Handler.java:790)
08-06 20:28:30.257  5493  5493 E AndroidRuntime:        at android.os.Handler.dispatchMessage(Handler.java:99)
08-06 20:28:30.257  5493  5493 E AndroidRuntime:        at android.os.Looper.loop(Looper.java:164)
08-06 20:28:30.257  5493  5493 E AndroidRuntime:        at android.app.ActivityThread.main(ActivityThread.java:7000)
08-06 20:28:30.257  5493  5493 E AndroidRuntime:        at java.lang.reflect.Method.invoke(Native Method)
08-06 20:28:30.257  5493  5493 E AndroidRuntime:        at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:441)
08-06 20:28:30.257  5493  5493 E AndroidRuntime:        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1408)
08-06 20:28:30.257  5493  5493 E SignalUncaughtException: 
08-06 20:28:30.257  5493  5493 E SignalUncaughtException: java.lang.NullPointerException: sensorManager.getDefault…or(Sensor.TYPE_PROXIMITY) must not be null
08-06 20:28:30.257  5493  5493 E SignalUncaughtException:       at org.thoughtcrime.securesms.components.voice.VoiceNoteProximityWakeLockManager.<init>(VoiceNoteProximityWakeLockManager.kt:39)
08-06 20:28:30.257  5493  5493 E SignalUncaughtException:       at org.thoughtcrime.securesms.components.voice.VoiceNoteMediaController$ConnectionCallback.onConnected(VoiceNoteMediaController.java:280)
08-06 20:28:30.257  5493  5493 E SignalUncaughtException:       at android.support.v4.media.MediaBrowserCompat$ConnectionCallback$StubApi21.onConnected(MediaBrowserCompat.java:669)
08-06 20:28:30.257  5493  5493 E SignalUncaughtException:       at android.support.v4.media.MediaBrowserCompatApi21$ConnectionCallbackProxy.onConnected(MediaBrowserCompatApi21.java:103)
08-06 20:28:30.257  5493  5493 E SignalUncaughtException:       at android.media.browse.MediaBrowser$6.run(MediaBrowser.java:603)
08-06 20:28:30.257  5493  5493 E SignalUncaughtException:       at android.os.Handler.handleCallback(Handler.java:790)
08-06 20:28:30.257  5493  5493 E SignalUncaughtException:       at android.os.Handler.dispatchMessage(Handler.java:99)
08-06 20:28:30.257  5493  5493 E SignalUncaughtException:       at android.os.Looper.loop(Looper.java:164)
08-06 20:28:30.257  5493  5493 E SignalUncaughtException:       at android.app.ActivityThread.main(ActivityThread.java:7000)
08-06 20:28:30.257  5493  5493 E SignalUncaughtException:       at java.lang.reflect.Method.invoke(Native Method)
08-06 20:28:30.257  5493  5493 E SignalUncaughtException:       at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:441)
08-06 20:28:30.257  5493  5493 E SignalUncaughtException:       at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1408)
```

</details>

**Comment by cody-signal** [comment]
Sorry you're hitting this. We have a fix coming already in 5.20.2 that should be rolling out soon.

**Comment by link2xt** [comment]
I already reinstalled to non-beta to get it working, feel free to close if it's fixed.

> We have a fix coming already in 5.20.2 that should be rolling out soon.

The latest commit is 80e1b2c843ce24442ebe669e888335c5cac2b68b (Bump version to 5.20.1), does it mean the fix is not published to GitHub yet?

**Comment by cody-signal** [comment]
Correct. I'm in the process of publishing it now.

**Comment by link2xt** [comment]
Fixed in de2c7d38bfeada998627594f85821b85215ffb43
